app/actions/project.js

```diff
@@ -13,9 +13,10 @@ export const ADD_REQUEST = 'ADD_REQUEST';
 export const UPDATE_REQUEST = 'UPDATE_REQUEST';
 export const DELETE_REQUEST = 'DELETE_REQUEST';
 export const EXECUTE_REQUEST = 'EXECUTE_REQUEST';
+export const LOG_REQUEST = 'LOG_REQUEST';
 
 export const Request = Record({
-  id: undefined,
+  id: randomId(),
```

Comment:
This is causing the regression I mentioned in my previous comment. The `RequestEditor` checks if a request can be deleted by checking if it has a `projectId` and `id` set. Because of this change every `Request` created will have an `id`, even if it's not saved yet.

What was the reason to change this? I think it makes sense to only assign an id when you actually persist it.